openfisca_us/parameters/states/ma/tax/income/ald/disallowed.yaml

```diff
@@ -0,0 +1,14 @@
+description: Above-the-line deductions disallowed in Massachusetts income tax.
+values:
+  2001-01-01:
+    - loss_ald
+    - ira_contributions
+    - sep_simple_qualified_plan_contributions
+    - early_withdrawal_penalty
+metadata:
+  unit: variable
+  name: ma_ald_disallowed
+  label: MA Above-the-line deductions disallowed
+  reference:
+    - title: Mass. General Laws c.62 § 2(d)(1)(A)
```

Comment:
Isn't it § 2(a)(2)?

Could you also leave comments on each line with the subsection? e.g. IRA is (F).

Reply:
Added comments - I think it is actually (d)(1)(A)